sympy/integrals/intpoly.py

```diff
@@ -0,0 +1,796 @@
+from __future__ import print_function, division
```

Comment:
Let's add a module docstring describing what the module is for and what it does. Doesn't have to be long, just so that when somebody opens it, it is clear what it does.